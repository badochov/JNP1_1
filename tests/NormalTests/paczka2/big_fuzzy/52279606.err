Error in line 25: eVS4AMvuz A954 219,08
Error in line 27: TdA3cLVcg A954 20,8
Error in line 39: S416 S694 89,9
Error in line 72: TtTnK6 A954 12,0
Error in line 59: sd44 S969 264,3
Error in line 78: sd44 A954 89,2
Error in line 79: sd44 S416 68,2
Error in line 71: 3okw S416 85,1
Error in line 80: sd44 A954 75,3
Error in line 90: eVS4AMvuz S969 57,1
Error in line 93: sd44 S969 93,3
Error in line 97: TdA3cLVcg S969 5,0
Error in line 103: TdA3cLVcg S694 82,7
Error in line 109: 3okw S694 94,p
Error in line 121: eVS4AMvuz S416 74,7
Error in line 123: FCMlw1x5E2 S416 2,4
Error in line 129: FCMlw1x5E2 A954 20,7
Error in line 122: 3okw S694 167,8
Error in line 152: TdA3cLVcg S694 38,2
Error in line 149: eVS4AMvuz A954 3,9
Error in line 160: 3okw A954 58,5
Error in line 174: sd44 S969 37,9
Error in line 188: eVS4AMvuz A954 6,91
Error in line 187: sd44 A954 4,7
Error in line 199: eVS4AMvuz S416 468,9
Error in line 200: eVS4AMvuz S969 100,0
Error in line 205: 3okw S694 89,3
Error in line 204: eVS4AMvuz A954 8,8
Error in line 213: sd44 S694 74,8
Error in line 210: TdA3cLVcg A954 83,8
Error in line 221: TdA3cLVcg S694 21,7
Error in line 225: 3okw S969 34,1
Error in line 230: sd44 S694 6,0
Error in line 245: TtTnK6 S416 78,7
