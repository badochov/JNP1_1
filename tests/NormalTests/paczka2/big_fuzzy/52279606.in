S416 S969 97,5
?eVS4AMvuz
FCMlw1x5E2 S694 90,2
eVS4AMvuz S694 17,1
eVS4AMvuz S694 5,3
?
TtTnK6 A954 15,8
TdA3cLVcg S416 76,7
sd44 A954 38,3
S416 S969 421,2
3okw S969 58,7

TtTnK6 A954 1,3
S416 S416 42,2
sd44 A954 140,2
TdA3cLVcg S416 204,0
3okw S969 77,5

eVS4AMvuz A954 95,0
?
eVS4AMvuz A954 55,1
?TtTnK6
TtTnK6 S694 70,6
S416 S416 80,6
eVS4AMvuz A954 219,08
eVS4AMvuz A954 92,4
TdA3cLVcg A954 20,8
?eVS4AMvuz
?
FCMlw1x5E2 S694 260,7

eVS4AMvuz A954 125,7
TdA3cLVcg S416 81,4
TtTnK6 S694 307,1
TdA3cLVcg S416 295,5
TdA3cLVcg S969 64,5
?

S416 S694 89,9
?
TtTnK6 A954 74,8
TdA3cLVcg S969 69,7
eVS4AMvuz A954 41,0
S416 A954 70,5
eVS4AMvuz A954 100,9
sd44 S694 1,3
FCMlw1x5E2 S416 8,5
S416 A954 331,4
S416 S416 7,8
3okw S969 94,4
TtTnK6 A954 42,8
sd44 S694 4,0
sd44 S969 43,8
3okw S969 2696,0
3okw S416 89,5
S416 S416 27,5
sd44 S969 88,7
3okw S416 90,3
sd44 S969 264,3
3okw S416 154,4
FCMlw1x5E2 S416 28,5
FCMlw1x5E2 S969 57,2
S416 A954 25,6
?S416
3okw S416 36,2
S416 A954 9,0

FCMlw1x5E2 S969 163,0
S416 S416 75,2

3okw S416 85,1
TtTnK6 A954 12,0
TtTnK6 S969 61,2
?
TdA3cLVcg S694 68,4
TdA3cLVcg S694 255,8
S416 S416 238,4
sd44 A954 89,2
sd44 S416 68,2
sd44 A954 75,3
3okw S969 73,1
3okw S969 30,7
3okw S694 13,6
sd44 S969 32,0
TtTnK6 S969 115,1
sd44 S969 12,2
?A954
3okw S694 34,0
TdA3cLVcg S969 22,3
eVS4AMvuz S969 57,1
S416 S416 28,1
eVS4AMvuz A954 49,5
sd44 S969 93,3
TdA3cLVcg S969 7,0
sd44 S694 12,6
3okw S694 84,0
TdA3cLVcg S969 5,0
S416 S416 71,4
eVS4AMvuz A954 42,1
FCMlw1x5E2 S416 15,5
FCMlw1xEE2 S416 39,1

TdA3cLVcg S694 82,7
3okw S694 193,2
TdA3cLVcg S416 74,2
FCMlw1x5E2 S416 4,1
eVS4AMvuz S694 22,3
sd44 S694 14,0
3okw S694 94,p
TdA3cLVcg S416 144,7
TtTnK6 S416 52,2
?
eVS4AMvuz S694 48,3
eVS4AMvuz S416 49,4
TtTnK6 S416 101,8
?S969
TtTnK6 S969 67,6
eVS4AMvuz S416 188,2
?
TtTnK6 S969 254,9
eVS4AMvuz S416 74,7
3okw S694 167,8
FCMlw1x5E2 S416 2,4
TdA3cLVcg S694 67,6
eVS4AMvuz S694 21,4
S416 S416 6,0
TdA3cLVcg S694 230,5
eVS4AMvuz S694 53,5
FCMlw1x5E2 A954 20,7
eVS4AMvuz S969 37,1
sd44 S416 37,8
eVS4AMvuz S969 76,1
S416 S416 1,0
sd44 S416 19,7
FCMlw1x5E2 S969 62,3
TdA3cLVcg S416 49,8
3okw A954 72,9
?
TdA3cLVcg S416 39,3
3okw A954 136,3
FCMlw1x5E2 S969 10,0

TtTnK6 A954 22,9
FCMlw1x5E2 S416 17,2
FCMlw1x5E2 S416 76,5
TdA3cLVcg S694 31,1
sd44 A954 86,5
TdA3cLVcg S694 130,6
eVS4AMvuz A954 3,9
TtTnK6 A954 51,3
?FCMlw1x5E2
TdA3cLVcg S694 38,2
TtTnK6 A954 46,7
?TtTnK6
TdA3cLVcg A954 54,3
TtTnK6 A954 52,8
eVS4AMvuz S416 37,8
TdA3cLVcg A954 110,8
sd44 A954 52,0
3okw A954 58,5
sd44 A954 84,5
sd44 A954 138,6
3okw S694 26,9
FCMlw1x5E2 S694 82,8
TdA3cLVcg S969 98,2
3okw S694 62,7
FCMlw1x5E2 S694 404,1
TdA3cLVcg S969 164,0

?
FCMlw1x5E2 S694 96,0
?S969
S416 A954 2,6
sd44 S969 37,9

?

FCMlw1x5E2 S694 391,7
?S969
sd44 A954 91,8
S416 A954 6,0
sd44 A954 70,6
sd44 A954 284,5
sd44 A954 2,0
?S416
S416 S416 59,8
sd44 A954 4,7
eVS4AMvuz A954 6,91
sd44 S969 99,1
eVS4AMvuz S416 98,5
FCMlw1x5E2 S969 90,1
S416 S416 71,2
FCMlw1x5E2 S969 213,5
TtTnK6 A954 36,3
sd44 S969 91,2
?sd44
S416 S416 46,3
S416 S416 112,0
eVS4AMvuz S416 468,9
eVS4AMvuz S969 100,0
?S969
TtTnK6 A954 42,8
TtTnK6 S694 62,3
eVS4AMvuz A954 8,8
3okw S694 89,3
3okw S969 1,9
3okw S969 33,9

?
TdA3cLVcg A954 83,8
3okw S969 2,2
eVS4AMvuz S969 78,6
sd44 S694 74,8
TtTnK6 S694 181,9
3okw S969 7,7
sd44 A954 26,0
3okw A954 7,1
eVS4AMvuz S969 268,8
3okw A954 32,6
?S694
TdA3cLVcg S694 21,7
FCMlw1x5E2 A954 71,4
FCMlw1x5E2 A954 96,4
TdA3cLVcg S416 5,8
3okw S969 34,1
sd44 A954 44,7
3okw A954 84,8
FCMlw1x5E2 S694 20,4
eVS4AMvuz S969 70,6
sd44 S694 6,0
?
eVS4AMvuz S969 236,5
sd44 S969 98,7
eVS4AMvuz S416 36,0
eVS4AMvuz S416 88,1

3okw A954 229,8
eVS4AMvuz S416 385,9
sd44 S969 33,6

S416 S416 91,1
TdA3cLVcg S416 25,6
FCMlw1x5E2 S694 23,3
?eVS4AMvuz
TtTnK6 S416 78,7
S416 S416 358,0
TtTnK6 A954 19,4
3okw S694 60,0
TtTnK6 A954 37,6
