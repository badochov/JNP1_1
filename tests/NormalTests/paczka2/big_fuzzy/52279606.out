eVS4AMvuz S 11,8
S694 11,8
3okw S 18,8
S416 S 323,7
TdA3cLVcg S 127,3
TtTnK6 A 14,5
eVS4AMvuz S 11,8
sd44 A 101,9
S416 127,3
S694 11,8
A954 116,4
S969 342,5
TtTnK6 A 14,5
eVS4AMvuz A 39,9 S 11,8
3okw S 18,8
S416 S 362,1
TdA3cLVcg S 127,3
TtTnK6 A 14,5
eVS4AMvuz A 39,9 S 11,8
sd44 A 101,9
S416 165,7
S694 11,8
A954 156,3
S969 342,5
3okw S 18,8
FCMlw1x5E2 S 170,5
S416 S 362,1
TdA3cLVcg S 341,4
TtTnK6 A 14,5 S 236,5
eVS4AMvuz A 73,2 S 11,8
sd44 A 101,9
S416 379,8
S694 418,8
A954 189,6
S969 342,5
3okw S 18,8
FCMlw1x5E2 S 170,5
S416 S 362,1
TdA3cLVcg S 341,4
TtTnK6 A 14,5 S 236,5
eVS4AMvuz A 73,2 S 11,8
sd44 A 101,9
S416 379,8
S694 418,8
A954 189,6
S969 342,5
S416 A 260,9 S 381,8
S416 420,3
3okw S 2739,4
FCMlw1x5E2 S 296,3
S416 A 277,5 S 381,8
TdA3cLVcg S 346,6
TtTnK6 A 46,5 S 236,5
eVS4AMvuz A 133,1 S 11,8
sd44 A 101,9 S 47,6
S416 538,5
S694 421,5
A954 559,0
S969 3100,0
A954 559,0
3okw S 2911,4
FCMlw1x5E2 S 307,7
S416 A 277,5 S 588,3
TdA3cLVcg S 619,8
TtTnK6 A 46,5 S 290,4
eVS4AMvuz A 140,5 S 11,8
sd44 A 101,9 S 68,8
S416 826,9
S694 739,9
A954 566,4
S969 3231,4
S969 3231,4
3okw S 2911,4
FCMlw1x5E2 S 307,7
S416 A 277,5 S 588,3
TdA3cLVcg S 619,8
TtTnK6 A 46,5 S 340,0
eVS4AMvuz A 140,5 S 176,6
sd44 A 101,9 S 68,8
S416 1015,3
S694 765,9
A954 566,4
S969 3231,4
3okw S 2911,4
FCMlw1x5E2 S 307,7
S416 A 277,5 S 593,3
TdA3cLVcg S 782,7
TtTnK6 A 46,5 S 527,3
eVS4AMvuz A 140,5 S 247,7
sd44 A 101,9 S 86,9
S416 1038,4
S694 960,9
A954 566,4
S969 3457,7
FCMlw1x5E2 S 419,3
TtTnK6 A 74,9 S 527,3
3okw A 63,4 S 2947,2
FCMlw1x5E2 S 740,6
S416 A 277,5 S 593,3
TdA3cLVcg A 56,5 S 958,5
TtTnK6 A 81,0 S 527,3
eVS4AMvuz A 140,5 S 247,7
sd44 A 190,5 S 86,9
S416 1108,2
S694 1417,5
A954 809,4
S969 3575,8
S969 3575,8
3okw A 63,4 S 2947,2
FCMlw1x5E2 S 740,6
S416 A 277,5 S 593,3
TdA3cLVcg A 56,5 S 958,5
TtTnK6 A 81,0 S 527,3
eVS4AMvuz A 140,5 S 247,7
sd44 A 190,5 S 86,9
S416 1108,2
S694 1417,5
A954 809,4
S969 3575,8
S969 3575,8
S416 A 280,9 S 593,3
S416 1108,2
sd44 A 494,2 S 94,8
S969 3707,1
3okw A 63,4 S 2979,2
FCMlw1x5E2 S 1159,7
S416 A 280,9 S 670,4
TdA3cLVcg A 56,5 S 958,5
TtTnK6 A 87,5 S 527,3
eVS4AMvuz A 140,5 S 308,4
sd44 A 494,2 S 94,8
S416 1246,0
S694 1713,2
A954 1123,0
S969 3739,1
S694 1832,8
3okw A 88,9 S 2984,7
FCMlw1x5E2 A 25,0 S 1159,7
S416 A 280,9 S 670,4
TdA3cLVcg A 56,5 S 958,5
TtTnK6 A 87,5 S 646,9
eVS4AMvuz A 140,5 S 498,6
sd44 A 512,9 S 94,8
S416 1246,0
S694 1832,8
A954 1192,2
S969 3934,8
eVS4AMvuz A 140,5 S 716,6
